nmagent/client_test.go

```diff
@@ -675,6 +675,15 @@ func TestGetHomeAz(t *testing.T) {
 			},
 			true,
 		},
+		{
+			"404 from NMA",
+			nmagent.AzResponse{},
+			"/machine/plugins?comp=nmagent&type=GetHomeAz%2Fapi-version%2F1",
+			map[string]interface{}{
+				"httpStatusCode": "404",
+			},
+			true,
```

Comment:
unrelatd to this PR but I don't believe 
if gotPath != test.expPath { on line 717 is adding any value. Why would NMA API path be different?

Reply:
Removed